src/views/domain-page/domain-page-content/domain-page-content.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import decodeUrlParams from '@/utils/decode-url-params';
 
-export default async function DomainPageContent() {
+import {
+  type DomainPageContentParams,
+  type Props,
+} from './domain-page-content.types';
+
+export default async function DomainPageContent(props: Props) {
+  const decodedParams = decodeUrlParams(
+    props.params
+  ) as DomainPageContentParams;
   // Add a switch case here, and depending on what domainTab is, return the corresponding page content
   // Otherwise just throw a 404: https://nextjs.org/docs/pages/api-reference/functions/get-server-side-props#notfound
-  return <div>Placeholder for Domain Page content</div>;
```

Comment:
Make sure to use the correct way when adding the not found error, since we are using the app router. This is the docs for it
https://nextjs.org/docs/app/api-reference/functions/not-found